src/__tests__/proposalVersions.int.test.ts

```diff
@@ -18,15 +18,15 @@ const agent = request.agent(app);
 
 const createTestBaseFields = async () => {
 	await createBaseField({
-		label: 'First Name',
-		description: 'The first name of the applicant',
+		defaultLabel: 'First Name',
```

Comment:
PLEASE do not kill me for saying this, and also feel free to ignore this thought...

We could just leave these as `label` and `description`.

Default is, in some sense, implied / unnecessary modifier.  A base field has a label and description, and then it has localizations for that label and description <-- that is the reality and doesn't need the word default to make sense.

Smaller point, but default could be slightly confusing if interpreted outside of the context of localization -- are we saying there are label / description overrides from a business logic sense?

A benefit of not changing this is that it would make this PR no longer be a breaking change, which means the frontend would continue to work without any modification.

It would also mean this PR is much smaller since many of the lines are renames of these fields.

---

Just food for thought...

Reply:
@slifty I would never make plans to kill a colleague over a PR review in any way that could be traced back to me...

No I agree, especially per our discussion this can and should be dropped!